textcounter.js

```diff
@@ -21,12 +21,14 @@
         base.init = function() {
             base.options = $.extend({}, $.textcounter.defaultOptions, options);
 
+            var counterElementId = base.options.counterId ?? base.el.id  + '_counter';
```

Comment:
My only concern with setting it this way is if there are multiple instances on a page, with an empty ID, then this would create duplicate IDs. Perhaps something like jQuery UIs `uniqueId` function behavior could be duplicated here: https://github.com/jquery/jquery-ui/blob/1be45388176080418aef84f3c98fda2c10fcdc42/ui/unique-id.js#L43